GVFS/GVFS/CommandLine/GVFSVerb.cs

```diff
@@ -1101,29 +1098,12 @@ private GVFSEnlistment CreateEnlistment(string enlistmentRootPath, GitAuthentica
                     this.ReportErrorAndExit("Error: " + GVFSConstants.GitIsNotInstalledError);
                 }
 
-                string hooksPath = null;
-                if (GVFSPlatform.Instance.UnderConstruction.RequiresDeprecatedGitHooksLoader)

```

Comment:
It looks like this was the only place this `RequiresDeprecatedGitHooksLoader ` was being used.  Can you remove it?